src/db.rs

```diff
@@ -1,12 +1,9 @@
 pub mod schema;
 
 use diesel::prelude::*;
-use dotenvy::dotenv;
 use std::env;
 
 pub fn establish_connection() -> SqliteConnection {
-    dotenv().ok();
```

Comment:
Are you assuming that the developer has direnv enabled?

Reply:
I do if you're developing with flake, no?